Rakefile

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env rake
+
+begin
```

Comment:
Having this `Rakefile` at the top-level of the vagrant directory seems to clutter the project. Is it possible to move this file into `spec`?